packages/entities/entities-shared/sandbox/pages/ConfigCardDisplayPage.vue

```diff
@@ -9,6 +9,7 @@
 
     <h2>Format: JSON</h2>
     <ConfigCardDisplay
+      :config="konnectConfig"
       fetcher-url="https://cloud.konghq.com/us/gateway-manager/91e192e0-5981-4662-a37d-7b24272c9da3/routes/0af86198-9822-46e0-9028-47b173caf4aa"
```

Comment:
Even though it's on the sandbox, we do have enough data `controlPlaneId` and `entityId` to generate this URL

Reply:
Updated